scripts/setup-helper-functions.sh

```diff
@@ -130,6 +130,10 @@ function cmake_install {
   CPU_TARGET="${CPU_TARGET:-avx}"
   COMPILER_FLAGS=$(get_cxx_flags $CPU_TARGET)
 
+  if [ "${CPU_TARGET}" == "arm64" ]; then
```

Comment:
Naveen, have you verified if the CMAKE_PREFIX_PATH and CMAKE_INSTALL_PREFIX are also "/opt/homebrew" on other OS? It would be safer to check both CPU and OS here.

Reply:
I did not check. In the same file, it is clear that arm64 target type is only for Apple M1.

Reply:
You can have arm64 with Linux like AWS Graviton. I see that we do not handle this under the Linux OS above.
But let's do the right thing here by adding `"$OS" == "Darwin"`.